src/test/java/org/eclipse/pass/schema/SchemaInstanceTest.java

```diff
@@ -0,0 +1,121 @@
+            + "                    \"bar\": {\"$ref\": \"two.json#/definitions/form/properties/foo\"},\r\n"
+            + "                    \"baz\": \"value\"\r\n" + "                }\r\n" + "            }\r\n"
+            + "        }\r\n" + "    }";
+
+    String four = "{\r\n" + "        \"$id\": \"http://example.org/schemas/four.json\",\r\n"
+            + "        \"definitions\": {\r\n" + "            \"form\": {\r\n" + "                \"properties\": {\r\n"
+            + "                    \"foo2\": {\"$ref\": \"one.json#/definitions/form/properties/foo\"},\r\n"
+            + "                    \"bar2\": {\"$ref\": \"two.json#/definitions/form/properties/foo\"},\r\n"
+            + "                    \"baz\": \"value\"\r\n" + "                }\r\n" + "            }\r\n"
+            + "        }\r\n" + "    }";
+
+    String five = "{\r\n" + "        \"$id\": \"http://example.org/schemas/five.json\",\r\n"
+            + "        \"definitions\": {\r\n" + "            \"form\": {\r\n" + "                \"properties\": {\r\n"
+            + "                    \"one\": 1,\r\n" + "                    \"two\": 2\r\n" + "                }\r\n"
+            + "            }\r\n" + "        }\r\n" + "    }";
+
+    String six = "{\r\n" + "        \"$id\": \"http://example.org/schemas/six.json\",\r\n"
+            + "        \"definitions\": {\r\n" + "            \"form\": {\r\n" + "                \"properties\": {\r\n"
+            + "                    \"one\": 1\r\n" + "                }\r\n" + "            }\r\n" + "        }\r\n"
+            + "    }";
+
+    String seven = "{\r\n" + "        \"$id\": \"http://example.org/schemas/seven.json\"\r\n" + "    }";
+
+    @SuppressWarnings("unchecked")
+    @Test
+    void testSort() throws JsonMappingException, JsonProcessingException {
+        SchemaInstance schema_one = new SchemaInstance(map.readTree(one));
+        SchemaInstance schema_two = new SchemaInstance(map.readTree(two));
+        SchemaInstance schema_three = new SchemaInstance(map.readTree(three));
+        SchemaInstance schema_four = new SchemaInstance(map.readTree(four));
+        SchemaInstance schema_five = new SchemaInstance(map.readTree(five));
+        SchemaInstance schema_six = new SchemaInstance(map.readTree(six));
+        SchemaInstance schema_seven = new SchemaInstance(map.readTree(seven));
+        ArrayList<SchemaInstance> toSort = new ArrayList<SchemaInstance>(Arrays.asList(schema_five, schema_two,
+                schema_seven, schema_one, schema_six, schema_three, schema_four));
+        ArrayList<SchemaInstance> expected = new ArrayList<SchemaInstance>(Arrays.asList(schema_one, schema_two,
+                schema_three, schema_four, schema_five, schema_six, schema_seven));
+        Collections.sort(toSort);
+        toSort.forEach(str -> {
+            System.out.println(str.getName());
```

Comment:
Fine for debugging but these types of print statements in tests should be removed eventually.